app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -42,7 +42,12 @@ def show
     end
 
     def submit5103
-      res = service.submit5103(@current_user, params[:id])
+      # Log if the user doesn't have a file number
+      # NOTE: We are treating the BIRLS ID as a substitute
+      # for file number here
+      ::Rails.logger.info('[5103 Submission] No file number') if @current_user.birls_id.nil?
```

Comment:
Moving this from `lib/lighthouse/benefits_claims/service.rb` so that I don't have to pass the user to the `submit5103` method anymore